tests/functions.test.js

```diff
@@ -8,13 +8,14 @@ chai.use(sinonChai);
 
 describe('functions', () => {
   let testArray = [];
-  beforeEach = () => {
+  const each = cases.each;
```

Comment:
Pro tip: you can use destructuring here:

`const { each } = cases;`

Reply:
Thanks so much for your suggestions. I will modify the codes as instructed.